src/main/java/duke/task/TaskType.java

```diff
@@ -0,0 +1,8 @@
+package duke.task;
+
+/**
+ * Represents the type of task.
+ */
+public enum TaskType {
+    TODO, DEADLINE, EVENT
+}
```

Comment:
I like the abstraction of TaskType into a new class to represent different types of tasks instead of just storing it as a String somewhere.